app/bundles/LeadBundle/Tests/Model/LeadModelTest.php

```diff
@@ -722,6 +722,67 @@ public function testModifiedCompanies(): void
         $this->leadModel->modifyCompanies($lead, $companies);
     }
 
+    public function testImportLeadWithCompanyDataAndSkipIfExists(): void
+    {
+        $lead = new Lead();
+        $lead->setId(1);
+
+        $mockUserModel = $this->getMockBuilder(UserHelper::class)
+            ->disableOriginalConstructor()
+            ->getMock();
```

Comment:
Please try to not mock at all. The unit tests are terrible as it adds more burden for the maintenance. If you change the internal implementation of the class/methods then you have to update the unit tests too. And if the unit test is passing then it doesn't mean the feature works. It's very hard to understand as well.

In comparison, a functional test is pretty simple. You create some entities, flush them to the database, visit some endpoint with some data and assert what is returned. You can be sure that if the feature breaks the test will break too. If you refactor the code then you can leave the functional test intact and it still confirms if the feature works or not. It adds massive value with no burden.

BTW I spent several days refactoring unit tests lately just because the PHPUNIT minor version was upgraded. All unit tests. None of the functional tests needed to be touched.

But if you want to be a rebel and must mock, please do it in style 😎 
```suggestion
        $mockUserModel = $this->createMock(UserHelper::class);
```